dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java

```diff
@@ -25,305 +25,256 @@
+    public static final String FILE_KEY = "file";
 
-    public static final int DEFAULT_IDLE_TIMEOUT = 600 * 1000;
+    public static final String DUMP_DIRECTORY = "dump.directory";
 
-    public static final int DEFAULT_HEARTBEAT = 60 * 1000;
+    public static final String CLASSIFIER_KEY = "classifier";
 
-    public static final int DEFAULT_TIMEOUT = 1000;
+    public static final String VERSION_KEY = "version";
 
-    public static final int DEFAULT_CONNECT_TIMEOUT = 3000;
+    public static final String REVISION_KEY = "revision";
 
     /**
-     * public static final int DEFAULT_REGISTRY_CONNECT_TIMEOUT = 5000;
+     * package version in the manifest
      */
-    public static final int DEFAULT_RETRIES = 2;
+    public static final String RELEASE_KEY = "release";
 
-    public static final int DEFAULT_FAILBACK_TASKS = 100;
+    public static final int MAX_PROXY_COUNT = 65535;
+    // END common
 
-    public static final int DEFAULT_FAILBACK_TIMES = 3;
+    // BEGIN dubbo-remoting-api
+    public static final String PAYLOAD_KEY = "payload";
+    /**
+     * 8M
+     */
+    public static final int DEFAULT_PAYLOAD = 8 * 1024 * 1024;
 
-    public static final int MAX_PROXY_COUNT = 65535;
+    public static final String BUFFER_KEY = "buffer";
 
     /**
      * default buffer size is 8k.
      */
     public static final int DEFAULT_BUFFER_SIZE = 8 * 1024;
 
-    public static final Integer DEFAULT_METADATA_REPORT_RETRY_TIMES = 100;
-    public static final Integer DEFAULT_METADATA_REPORT_RETRY_PERIOD = 3000;
-    public static final Boolean DEFAULT_METADATA_REPORT_CYCLE_REPORT = true;
-
     public static final int MAX_BUFFER_SIZE = 16 * 1024;
 
     public static final int MIN_BUFFER_SIZE = 1 * 1024;
 
-    public static final String REMOVE_VALUE_PREFIX = "-";
+    public static final String CONNECT_TIMEOUT_KEY = "connect.timeout";
 
-    public static final String PROPERTIES_CHAR_SEPERATOR = "-";
+    public static final int DEFAULT_CONNECT_TIMEOUT = 3000;
 
-    public static final String GROUP_CHAR_SEPERATOR = ":";
+    public static final String HEARTBEAT_KEY = "heartbeat";
 
-    public static final String HIDE_KEY_PREFIX = ".";
+    public static final int DEFAULT_HEARTBEAT = 60 * 1000;
 
-    public static final String DEFAULT_KEY_PREFIX = "default.";
+    public static final String IDLE_TIMEOUT_KEY = "idle.timeout";
 
-    public static final String DEFAULT_KEY = "default";
+    public static final int DEFAULT_IDLE_TIMEOUT = 600 * 1000;
 
-    public static final String LOADBALANCE_KEY = "loadbalance";
+    public static final String ACCEPTS_KEY = "accepts";
 
-    /**
-     * key for router type, for e.g., "script"/"file",  corresponding to ScriptRouterFactory.NAME, FileRouterFactory.NAME
-     */
-    public static final String ROUTER_KEY = "router";
+    public static final int DEFAULT_ACCEPTS = 0;
 
-    public static final String CLUSTER_KEY = "cluster";
+    public static final String SERIALIZATION_KEY = "serialization";
```

Comment:
I think serialization can also be used as a constant alone. What do you think?

I think that a completely independent constant can be placed in its own module similar to the interface you added.
But constants that will be used by other modules can add SubConstants to the common module.
For example:
SERIALIZATION_KEY is used by other modules, but it belongs to the SERIALIZATION module. We can manage it by creating SerializationConstants in the common module, but the Serialization ID will only be used in the Serialization module. We can create Constants for management in the Serialization module as you do. 
What do you think?

Reply:
that's what I am thinking too. This is only step1, in step2, we will move them into separated classes, then in the later steps, we will evaluate if it is suitable to move into other module.